src/graphql/data/nft/NftBalance.ts

```diff
@@ -111,26 +113,54 @@ const nftBalanceQuery = graphql`
 type NftBalanceQueryAsset = NonNullable<
   NonNullable<NonNullable<NftBalanceQuery_nftBalances$data['nftBalances']>['edges']>[number]
 >
-//
-// export type TokenQueryData = NonNullable<TokenQuery$data['tokens']>[number]
+
+export function useLoadNftBalanceQuery(
+  ownerAddress?: string,
+  collectionAddress?: string | string[],
+  tokenId?: string
+): void {
+  const [, loadQuery] = useQueryLoader(nftBalanceQuery)
+  useEffect(() => {
+    if (ownerAddress) {
+      loadQuery({
+        ownerAddress,
+        filter: tokenId
+          ? { assets: [{ address: collectionAddress, tokenId }] }
+          : { addresses: Array.isArray(collectionAddress) ? collectionAddress : [collectionAddress] },
+        first: tokenId ? 1 : DEFAULT_WALLET_ASSET_QUERY_AMOUNT,
```

Comment:
not clear to me what the `first` means here

Reply:
basically, when would tokenId get passed in

Reply:
first means GraphQL will return the first `n` assets. When tokenId is passed in we expect only 1 item to return that corresponds to the given collection and tokenId, otherwise we expect to be grabbing the standard page size, ie first 25 items the user owns